charts/kserve-resources/README.md

```diff
@@ -66,6 +67,8 @@ $ helm install kserve oci://ghcr.io/kserve/charts/kserve --version v0.12.0-rc1
 | kserve.servingruntime.alibi.image | string | `"kserve/alibi-explainer"` |  |
 | kserve.servingruntime.art.defaultVersion | string | `"v0.12.0-rc1"` |  |
 | kserve.servingruntime.art.image | string | `"kserve/art-explainer"` |  |
+| kserve.servingruntime.huggingfaceserver.image | string | `"kserve/huggingfaceserver"` |  |
```

Comment:
Probably leftovers of a different PR

Reply:
Yeah I am fixing that there https://github.com/kserve/kserve/pull/3437